SPECS/sysbench/sysbench.spec

```diff
@@ -0,0 +1,166 @@
+Summary:        Scriptable database and system performance benchmark
+Name:           sysbench
+Version:        1.0.20
+Release:        1%{?dist}
+License:        GPLv2+
+Group:          Applications/System
+URL:            https://github.com/akopytov/sysbench/
+Source0:        https://github.com/akopytov/%{name}/archive/%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  automake
+BuildRequires:  libaio-devel
+BuildRequires:  libtool
+BuildRequires:  make
+BuildRequires:  mariadb-devel
+BuildRequires:  pkgconfig
+BuildRequires:  postgresql-devel
+BuildRequires:  python
```

Comment:
Can we change this build requirement to specifically reference `python3`?

Reply:
Tested, and change has been applied via git commit --amend!